pyproject.toml

```diff
@@ -24,10 +24,6 @@ python = "^3.6.2"
 aws-xray-sdk = "^2.8.0"
 fastjsonschema = "^2.14.5"
 boto3 = "^1.18"
```

Comment:
Could you add a comment here so we review jmespath state after 3.6 Lambda deprecation?

I think we can trust boto to not drop jmespath given how crucial it is for them, but I wanna be conservative as (1) it doesn't add any extra bytes at runtime, and (2) jmespath hasn't been updated in years

Reply:
After this we can merge it!